Data/DataModel/HelpMessage.cs

```diff
@@ -0,0 +1,62 @@
+﻿using System;
+using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+
+namespace ServerCore.DataModel
+{
+    /// <summary>
+    /// Message in help thread when a player asks for help.
+    /// </summary>
+    public class HelpMessage
+    {
+        [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
+        public int ID { get; set; }
+
+        /// <summary>
+        /// The date time (in UTC) of when the message was sent.
+        /// </summary>
+        public DateTime DateTimeInUtc { get; set; }
+
+        /// <summary>
+        /// The puzzle id that this message is for.
+        /// </summary>
+        [Required]
+        public int PuzzleID { get; set; }
```

Comment:
Are there use cases where there is no puzzle? E.g. 2-way communications about pizza etc.

Reply:
If so, should that have a different data model?

Reply:
There are two separate sets in the puzzle context now